packages/paste-website/src/components/tokens-list/token-card/index.tsx

```diff
@@ -0,0 +1,132 @@
+  comment: string;
+}
+export const TokenCard: React.FC<TokenCardProps> = ({category, name, value, comment}) => {
+  const tooltipState = useTooltipState();
+  const [tooltipText, setTooltipText] = React.useState('Copy token name');
+  // Prevents tooltip being visible on first render due to reakit positioning bug code
+  const isFirstRender = React.useRef(true);
+  const clipboard = useClipboard({copiedTimeout: 2000});
+  const camelCaseName = camelCase(name.replace('$', ''));
+
+  const handleCopyName = React.useCallback(() => {
+    clipboard.copy(camelCaseName);
+  }, [camelCaseName]);
+
+  React.useEffect(() => {
+    setTooltipText(clipboard.copied ? 'Copied!' : 'Copy token name');
+  }, [clipboard.copied]);
+
+  // Reakit has a bug where the tooltip doesn't recalc position on content changes
+  // This is a workaround until we upgrade to Ariakit with Floating UI fixes
+  // https://github.com/twilio-labs/paste/discussions/2037
+  React.useEffect(() => {
+    // This prevents the tooltip from showing up on mount
+    if (isFirstRender.current) {
+      isFirstRender.current = false;
+      return;
+    }
+    tooltipState.hide();
+    setTimeout(() => tooltipState.show(), 0);
+  }, [tooltipText]);
+
+  return (
+    <Box
+      key={name}
+      display="flex"
+      alignItems="center"
+      backgroundColor="colorBackgroundBody"
+      borderColor="colorBorderWeaker"
+      borderWidth="borderWidth10"
+      borderStyle="solid"
+      borderRadius="borderRadius30"
+      minHeight="sizeSquare170"
+    >
+      <Box margin="space40" width="sizeSquare200" justifyContent="center" alignItems="center" display="flex">
+        <PreviewComponent category={category} name={camelCaseName} value={value} />
+      </Box>
+      <Box
+        paddingY="space60"
+        paddingLeft="space70"
+        paddingRight="space70"
+        borderLeftColor="colorBorderWeaker"
+        borderLeftWidth="borderWidth10"
+        borderLeftStyle="solid"
+        width="100%"
+      >
+        <Box display={['block', 'flex']} columnGap="space110">
+          <Box marginBottom={['space40', 'space0']} flexGrow={1}>
+            <Box display="flex" alignItems="center" flexWrap="wrap">
+              <Text
+                as="span"
+                fontFamily="fontFamilyCode"
+                fontWeight="fontWeightBold"
+                fontSize="fontSize30"
+                lineHeight="lineHeight40"
+                marginRight="space20"
+                wordBreak="break-word"
+              >
+                {camelCaseName}
+              </Text>
+              <Tooltip text={tooltipText} state={tooltipState}>
+                <Button variant="secondary_icon" size="icon_small" onClick={handleCopyName}>
+                  <CopyIcon decorative />
+                  <span aria-live="polite">
+                    <ScreenReaderOnly>{clipboard.copied ? 'Copied token name!' : 'Copy token name'}</ScreenReaderOnly>
+                  </span>
+                </Button>
```

Comment:
issue(maybe): this is the screenshot from my chromatic comment. Notice the outline isn't truly a square, because we're applying column gap to the right on the icon due to the presence of screen reader text. Sure this is a known bug

<img width="228" alt="image" src="https://user-images.githubusercontent.com/368249/166806744-266afce2-fb3f-4e49-964a-b5c6af845023.png">